cudaextender/src/ungapped_xdrop_kernels.cuh

```diff
@@ -60,21 +60,50 @@ __global__ void compress_output(const int32_t* d_done,
                                 int num_hits);
 
 // Binary predicate for sorting the ScoredSegmentPairs
+struct scored_segment_pair_equal
+{
+    __host__ __device__ bool operator()(ScoredSegmentPair x, ScoredSegmentPair y)
```

Comment:
I would use `const ScoredSegmentPair&` here.

Reply:
Good point! I will change it.